### PR TITLE
Simplify linux doc

### PR DESCRIPTION
**Pull request by richlander**
Simplify the document. After this, we need to update some of the distro docs. For example, the Alpine doc doesn't describe how to install from the Alpine package manager.

Related to: https://github.com/dotnet/core/pull/7971

### CODE REVIEW

File: docs/core/install/linux.md [hunk 1]
```diff
@@ -14,55 +14,31 @@ ms.date: 11/08/2022
 > - [Install on macOS](macos.md)
 > - [Install on Linux](linux.md)
 
-This article details how to install .NET on various Linux distributions, manually and via a package manager. Typically, stable .NET versions are available in a package manager, and Preview versions are not.
-
-The latest version of .NET is 7.
+This article details how to install .NET on various Linux distributions, manually, via a package manager, or [containers](../docker/introduction.md#net-core-images).
```
Comment (gewarren) [inline docs/core/install/linux.md:17]:
```suggestion
This article details how to install .NET on various Linux distributions either manually, via a package manager, or via a [container](../docker/introduction.md#net-core-images).
```

**Comment by adegeo** [comment]
This should be done in #32596

File: docs/core/install/linux.md [hunk 2]
```diff
@@ -14,55 +14,31 @@ ms.date: 11/08/2022
...
-- [Red Hat Enterprise Linux](https://access.redhat.com/documentation/en-us/net/6.0)
-- [Ubuntu](https://packages.ubuntu.com/search?keywords=dotnet6)
-
-Microsoft collaborates with partners to ensure .NET works well on their Linux distributions. Support is provided by those distributions. You can still [open issues at dotnet/core](https://github.com/dotnet/core/issues) if you run into problems.
+## Packages
 
-## Microsoft packages
-
-.NET is also available via [packages.microsoft.com](https://packages.microsoft.com/).
+.NET is available in [official package archives](https://github.com/dotnet/core/blob/main/linux.md) for various Linux distributions and [packages.microsoft.com](https://packages.microsoft.com/).
```
Comment (adegeo) [inline docs/core/install/linux.md:30]: Dependent on dotnet/core#7971

File: docs/core/install/linux.md [hunk 3]
```diff
@@ -14,55 +14,31 @@ ms.date: 11/08/2022
...
 
+- [Alpine](linux-alpine.md)
```
Comment (adegeo) [inline docs/core/install/linux.md:32]: We need instructions added to #32596 for Alpine.

File: docs/core/install/linux.md [hunk 4]
```diff
@@ -14,55 +14,31 @@ ms.date: 11/08/2022
...
-You're encouraged to install .NET from the official archive for your distribution if it's available there, even if it's also available at packages.microsoft.com.
-
-## Other distributions
-
-Installation information is also provided for other distributions.
-
-- [Alpine](linux-alpine.md)
-- [Containers](../docker/introduction.md#net-core-images)
-- [Snap](linux-snap.md)
+.NET is [supported by Microsoft](https://github.com/dotnet/core/blob/main/microsoft-support.md) when downloaded from a Microsoft source. Best effort support is offered from Microsoft when downloaded from elsewhere. You can  open issues at [dotnet/core](https://github.com/dotnet/core) if you run into problems.
```
Comment (adegeo) [inline docs/core/install/linux.md:41]: Fix double space
```suggestion
.NET is [supported by Microsoft](https://github.com/dotnet/core/blob/main/microsoft-support.md) when downloaded from a Microsoft source. Best effort support is offered from Microsoft when downloaded from elsewhere. You can open issues at [dotnet/core](https://github.com/dotnet/core) if you run into problems.
```

File: docs/core/install/linux.md (hunk 2 above)
Comment (richlander) [reply docs/core/install/linux.md:30]: Merged now.

File: docs/core/install/linux.md (hunk 3 above)
Comment (richlander) [reply docs/core/install/linux.md:32]: We can add that in a separate PR.

**Review by adegeo — APPROVED** [review]

**Pull request merged**